test/e2e/network/netpol/model.go

```diff
@@ -41,6 +41,29 @@ type Model struct {
 	DNSDomain      string
 }
 
+// NewWindowsModel returns a model specific to windows testing.
+func NewWindowsModel(namespaces []string, podNames []string, ports []int32, dnsDomain string) *Model {
+	return NewModel(namespaces, podNames, ports, []v1.Protocol{v1.ProtocolTCP}, dnsDomain)
+}
+
+// GetProbeTimeoutSeconds returns a timeout for how long the probe should work before failing a check, and takes windows heuristics into account, where requests can take longer sometimes.
+func (m *Model) GetProbeTimeoutSeconds() int {
+	timeoutSeconds := 1
+	if framework.NodeOSDistroIs("windows") {
+		timeoutSeconds = 3
+	}
+	return timeoutSeconds
+}
+
+// GetWorkers returns the number of workers suggested to run when testing, taking windows heuristics into account, where parallel probing is flakier.
+func (m *Model) GetWorkers() int {
```

Comment:
if we make `NumberOfWorkers` and `NimeoutSeconds` part of the `Model` struct, so they are initialised directly from the `NewModel` or `NewWindowsModel`  you save the getters
But this is just up to you

Reply:
i think these will still be functions eventually bc we'll have different timeouts for udp and so on....  Windows on UDP is going to be its own animal :) :)

Reply:
No strong opinion, about the implementation but we should reflect that in comments, having the same tests with different parameters is not good, basically we are modifying the test behaviour to pass , that means that the feature is different in each platform

Reply:
ok, can we do that here https://github.com/kubernetes/kubernetes/issues/102334 ?